static/js/services/live-list.js

```diff
@@ -68,7 +69,17 @@ angular.module('inboxServices').factory('LiveListConfig',
           scope.simprintsTier = contact.simprints && contact.simprints.tierNumber;
           scope.dod = contact.date_of_death;
           if (contact.type !== 'person') {
-            scope.summary = $translate.instant('contact.primary_contact_name', { name: contact.contact });
+            if (Number.isInteger(contact.lastVisitedDate)) {
+              var now = new Date().getTime();
+              var oneMonthAgo = now - (30 * 24 * 60 * 60 * 1000);
+              if (contact.lastVisitedDate === 0 || contact.lastVisitedDate <= oneMonthAgo) {
+                scope.overdue = true;
+              }
+
+              scope.summary = $translate.instant('contact.lastVisitedDate', { date: relativeDayFilter(contact.lastVisitedDate, true) });
```

Comment:
It's bad form to call a filter from a service. Ideally we'd pull out the useful functions into the FormatDate service and call that instead, but I'll leave it up to you to decide whether to do this as part of 2.16.0 or we can fix it in a later refactor...

Reply:
Can you clarify what you mean by "bad form". The AngularJS documentation seems OK with it, but they don't talk expansively about it apart from explaining the naming convention.

Reply:
It feels to me like it's calling the wrong part of the stack, like in MVC the C calling code in the V. But I have no practical reason to dislike it and I can't find anything online to support my prejudice...